backend/api/middlewares/logger_middleware.py

```
@@ -18,5 +19,17 @@ async def dispatch(
         :return: Response from endpoint
         """
         # TODO:(Member) Finish implementing this method
-        response = await call_next(request)
-        return response
+        
+        start_time = perf_counter()
+        logger.info(f"{request.method} {request.url.path}")
+            
+        try: 
+            response = await call_next(request)
+            duration = perf_counter() - start_time
+            logger.info(f"{request.method} {request.url.path} {response.status_code} in {duration:.2f} seconds")
+            return response
+        except Exception as e:
+            duration = perf_counter() - start_time
+            logger.error(f"{request.method} {request.url.path} ERROR in {duration:.2f}s: {str(e)}")
+            raise
```

Could you raise a specific error either by checking if there are database error defined or by just using `ValueError("Error Description")` with a good string between the brackets to describe the error.